src/main/java/org/folio/dcb/config/props/SystemUserProperties.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.dcb.config.props;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@ConfigurationProperties("system-user")
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SystemUserProperties {
```

Comment:
I don't see the usage of this class anywhere. Can you please check

Reply:
done.
class removed.